2019_amsterdam.html

```diff
@@ -0,0 +1,170 @@
+    </nav>
+    <section class="bg-dark">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-8 col-lg-offset-2 text-center">
+                    <h1 class="section-heading">June 2019 Amsterdam</h1>
+                    <br />
+                    <h2>June 5-7 2019</h2>
+                    <p>CoreDev Hacking Meetings <em>(invite only)</em></p>
+                    <hr />
+                    <h4>June 8-9 2019</h4>
+                    <p><a href="https://breaking-bitcoin.com/">Breaking Bitcoin 2</a> (security conference on Bitcoin)
+                    </p>
+                    <hr />
+                    <h4>June 11-13 2019</h4>
+                    <p><a href="https://breaking-bitcoin.com/otherPages/2019/training.html">Bitcoin dev
+                            training & workshop</a> by Justin Moon,
+                        Stephane Roche and Mickael Folkson (similar to the amazing Bitcoin
+                        Edge, but much more hand-on)</p>
+                    <hr />
+                    <h4>June 14-16 2019</h4>
+                    <p><a href="https://blockchain-hackathon.com/">Chainhack hackathon</a> (150 participants)</p>
+                    <hr />
+                    <h4>June ?-? 2019</h4>
+                    <p>Breaking Bitcoin will also book a coworking space for the week, so
+                        anybody who wished to work together with fellow bitcoiners can do so
+                        for a few days. (location and dates TBD)</p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="bg-primary" id="info-milan">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-8 col-lg-offset-2 text-center">
+
+                    <h2 class="section-heading">Location</h2>
+                    <p>Amsterdam, Netherlands
+                        <br />
+                        <em>various locations</em>
+                    </p>
+                    <iframe
+                        src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d155959.95436057376!2d4.758540758964496!3d52.35456528430268!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x47c63fb5949a7755%3A0x6600fd4cb7c0af8d!2sAmsterdam%2C+Netherlands!5e0!3m2!1sen!2sus!4v1553280160036"
+                        width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section id="participants">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-8 col-lg-offset-2 text-center">
+
+                        <h2 class="section-heading">Sponsors</h2>
+                        <hr />
+                        <p><a href="https://building-on-bitcoin.com/"><img src="https://breaking-bitcoin.com/img/Logo_horiz_white_min.png" /></a></p>    
+                    <br /><br />
```

Comment:
Should this instead be the breaking bitcoin web site? https://breaking-bitcoin.com

Reply:
Ill confirm with Kevin.